unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -2066,6 +2066,21 @@ execute =
         _ -> Left $ showPatternHelp execute
     )
 
+saveExecuteResult :: InputPattern
+saveExecuteResult =
+  InputPattern
+    "add.run"
+    []
+    I.Visible
+    [(ZeroPlus, noCompletions)]
+    ( "`add.run name` adds to the codebase the result of the most recent `run` command"
+        <> "as `name`."
+    )
+    ( \case
+        [w] -> pure $ Input.SaveExecuteResultI (Name.unsafeFromString w)
```

Comment:
Should we display pattern help on `[""]` too maybe? It's unlikely that the user will enter it explicitly, but if they do it'll trigger an `error` call by `unsafeFromString`

Reply:
Do we allow passing empty strings as arguments? I think we may not allow it.

Reply:
I don't think we can pass empty strings currently.

Reply:
Ah, I thought Haskeline handled quotes and might pass something with whitespace, but that doesn't appear to happen here.